README.md

```diff
@@ -11,31 +11,31 @@ variables or on the command-line.
 
 This version of the container includes:
 
-- pgTAP 1.2.1 (via git at `56c591fc3bf6b2dba49e15739625916d49e3fc27`)
+- pgTAP 1.2.1 (via git at `96a7a416311ea5f2fa140f59cfdf7c7afbded17c`)
```

Comment:
Is pgTAP 1.2.1 a real release, or is it just the tip of master? I guess this is fall-out from their versioning.

Reply:
They haven’t released 1.2.1, but it has updates that are required to use newer versions. So I’ve configured it so we can either download (with a straight release) or git checkout (with a release and hash). That’s in `Maintenance.md`.